Phraseanet-production-client/src/components/ui/workzone/index.js

```diff
@@ -482,6 +482,18 @@ const workzone = (services) => {
         $('#basket-filter INPUT').change(function () {
             const sel = $(this).val();
             $(sel).toggleClass('hidden', !$(this).is(':checked'));
+            // save in user setting
+            $.ajax({
+                type: 'POST',
+                url: '/user/preferences/',
+                data: {
+                    prop: $(this).attr("data-prop"),
+                    value: $(this).is(':checked') ? 1 : 0
+                },
+                success: function (data) {
+                    return;
+                }
+            });
         });
     }
```

Comment:
During init, show / hide blocks depending on checkboxes. loop on $('#basket-filter INPUT')
This ensure we use the same algorithm on init / change. No risk to forget to replicate a js effect to the twig.